src/tests/webkit/WebKit_Memory_001.test.js

```diff
@@ -62,7 +62,7 @@ export default {
     let response = this.$data.read('monitorinfo')
     for (let i = 0; i < response.length; i++) {
       let plugin = response[i]
-      if (plugin.observable === constants.webKitBrowserPlugin) {
+      if (plugin.observable === constants.youTubePlugin) {
```

Comment:
why are we changing this to the youtubeplugin? We should be running this in webKitBrowser right?

Reply:
Sorry , This was a typo as I was checking the plugin which is not available. I will update this to web kit